src/Console/Infrastructure/Command/ListModulesCommand.php

```diff
@@ -21,12 +22,14 @@ final class ListModulesCommand extends Command
     protected function configure(): void
     {
         $this->setName('list:modules')
-            ->setDescription('Render all modules found');
+            ->setDescription('Render all modules found')
+            ->addArgument('filter', InputArgument::OPTIONAL, 'The filter path. For example "App/TestModule/TestSubModule');
```

Comment:
I would suggest this description to something simpler:
```suggestion
            ->addArgument('filter', InputArgument::OPTIONAL, 'Any filter to simplify the output');
```
What do you think, @JesusValera and @Tito-Kati ?